url/url_test.cpp

```diff
@@ -634,6 +637,105 @@ int main() {
         etest::expect_eq(*url, url::Url{.scheme = "a", .host = url::Host{.type = url::HostType::Opaque}});
     });
 
+    etest::test("Web Platform Tests", [] {
+        url::UrlParser p;
+
+        simdjson::ondemand::parser parser;
+
+        auto json = simdjson::padded_string::load("external/wpt/url/resources/urltestdata.json");
+
+        simdjson::ondemand::document doc = parser.iterate(json);
+
+        simdjson::ondemand::array arr = doc.get_array();
+
+        for (auto obj : arr) {
```

Comment:
`auto const &obj : arr`, or we'll copy every element in the array.

Reply:
Ah, right. You can leave it like this then. Maybe they've overloaded operator[] to something like lookup-or-create like std::map has.